util/util.py

```diff
@@ -269,3 +269,9 @@ def get_display_name_from_user(user):
             return user['name']
 
     return user['id']
+
+
+def is_valid_name(name):
```

Comment:
Is this used anywhere? if we got rid of `import *`, and I don't see existing mentions in this file, how is it used?

Reply:
It's still used in many places in the ChallengeHandler. You don't see a mention because we kept one wildcard import for now `from util.util import *` because there are too much functions from util being used in ChallgeneHandler.